services/streammanager/wunderkind/wunderkindmanager.go

```diff
@@ -0,0 +1,110 @@
+package wunderkind
+
+import (
+	"encoding/json"
+
+	"github.com/aws/aws-sdk-go/service/lambda"
+	jsoniter "github.com/json-iterator/go"
+
+	"github.com/rudderlabs/rudder-go-kit/awsutil"
+	"github.com/rudderlabs/rudder-go-kit/logger"
+	backendconfig "github.com/rudderlabs/rudder-server/backend-config"
+	"github.com/rudderlabs/rudder-server/cmd/rudder-cli/config"
```

Comment:
We need to use the cone from the go-kit.
```suggestion
	"github.com/rudderlabs/rudder-go-kit/config"
```

Reply:
addressed